.github/workflows/velox.yml

```diff
@@ -0,0 +1,54 @@
+name: Velox Nightly Build and Test
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  container:
+    runs-on: ubuntu-latest
```

Comment:
since there is a container specified -- would this still be useful (e.g. I imagine it's always running the linux in container? ~)

Reply:
Let me see if i can review it, i think this is supposed to be the vm in which the docker container runs..

Reply:
runs-on cant be removed its required ..